src/main/kotlin/io/github/mojira/arisa/infrastructure/jira/Functions.kt

```diff
@@ -202,12 +202,18 @@ fun addRestrictedComment(
 }
 
 fun createLink(context: Lazy<IssueUpdateContext>, getContext: (key: String) -> Lazy<IssueUpdateContext>, linkType: String, linkKey: String, outwards: Boolean) {
-    context.value.otherOperations.add {
-        runBlocking {
-            Either.catch {
-                context.value.jiraIssue.link(linkKey, linkType)
+    when {
```

Comment:
if :P

Reply:
facepalm.jpg
copied from TransferLinksModule tho

Reply:
"Always leave the code better than you found it" :D

Reply:
fixed by https://github.com/mojira/arisa-kt/pull/428/commits/ace0b9230685c383f2e88d17e4e5a004c4a69f2a